spec/index.html

```diff
@@ -1516,58 +1516,71 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
+              in |a| is equal the <a data-cite="INFRA#list-item">item</a>
+              at the corresponding index in |b|,
+              and both |a| and |b| have the same <a data-cite="INFRA#list-size">size</a>; or</li>
+            <li>if they are both <a data-cite="INFRA#ordered-map">maps</a> with equal <a data-cite="INFRA#entry">entries</a>
+              – meaning that each a <a data-cite="INFRA#entry">entry</a> in |a|
+              has an equivalent <a data-cite="INFRA#entry">entry</a> in |b|,
+              and both |a| and |b| have the same <a data-cite="INFRA#map-size">size</a>.</li>
+          </ul>
+      </dd> 
+
+      <dt id="JSON-mapping">The <a>lexical-to-value mapping</a></dt>
+      <dd>maps every element of the lexical space to the result of parsing it into a
+        <a>string</a>,
+        number (<a data-cite="XMLSCHEMA11-2#double">double</a>),
+        <a data-cite="INFRA#ordered-map">map</a>,
+        <a data-cite="INFRA#list">list</a>, or
+        literal value (<a data-cite="INFRA#boolean">`true`, `false`</a>, and <a data-cite="INFRA#nulls">null</a>).
+
         <ul>
-          <li>They MUST NOT contain any unnecessary whitespace.</li>
-          <li>Keys in objects MUST be ordered lexicographically.</li>
-          <li>Native Numeric values MUST be serialized according to
-            <a data-cite="ECMASCRIPT#sec-tostring-applied-to-the-number-type">Section&nbsp;7.1.12.1</a>
-            of [[ECMASCRIPT]].</li>
-          <li>Strings MUST be serialized with Unicode code points from <code class="codepoint">U+0000</code>
-            through <code class="codepoint">U+001F</code> using lower case hexadecimal Unicode notation
-            (`\uhhhh`) except for the set of predefined JSON control characters —
-            `U+0008`, `U+0009`, `U+000A`, `U+000C`, and `U+000D` — which SHOULD be
-            serialized as `\b`, `\t`, `\n`, `\f`, and `\r`, respectively. All
-            other Unicode characters SHOULD be serialized "as is", except
-            `U+005C` (`\`) and `U+0022` (`"`), which SHOULD be serialized as
-            `\\` and `\"`, respectively.</li>
+          <li>A <a href="RFC8259#section-4">JSON Object</a> is mapped to a <a data-cite="INFRA#ordered-map">map</a>
+            by transforming each object member into a <a data-cite="INFRA#map-entry">map entry</a>
+            with the <a data-cite="INFRA#map-key">key</a> taken from the member name and
+            <a data-cite="INFRA#map-value">value</a> taken by performing this mapping
+            to the member value.
+            <a data-cite="INFRA#map-entry">Map entries</a> are treated as being unordered.
+            </li>
+          <li>A <a href="RFC8259#section-5">JSON Array</a> is mapped to a <a data-cite="INFRA#list">list</a>
+            by performing this mapping on each array value.</li>
+          <li>A <a href="RFC8259#section-6">JSON Number</a> is mapped to
+            a <a data-cite="XMLSCHEMA11-2#double">double</a>.</li>
```

Comment:
If we need to be explicit on how the lexical representation is turned into a value, we can call out the paragraph in xsd:double that describes the mapping (In my view, this is implicit):

> A literal in the [·lexical space·](https://www.w3.org/TR/xmlschema-2/#dt-lexical-space) representing a decimal number d maps to the normalized value in the [·value space·](https://www.w3.org/TR/xmlschema-2/#dt-value-space) of double that is closest to d; if d is exactly halfway between two such values then the even value is chosen. This is the best approximation of d ([[Clinger, WD (1990)]](https://www.w3.org/TR/xmlschema-2/#clinger1990), [[Gay, DM (1990)]](https://www.w3.org/TR/xmlschema-2/#gay1990)), which is more accurate than the mapping required by [[IEEE 754-1985]](https://www.w3.org/TR/xmlschema-2/#ieee754).

Reply:
Isn't that from the old version of XML Schema datatypes?

Reply:
Yes, sorry for the old reference. The reference would be to the [Lexical Mapping](https://www.w3.org/TR/xmlschema11-2/#sec-lex-double) section and specifically the following:

> The [double](https://www.w3.org/TR/xmlschema11-2/#double) datatype is designed to implement for schema processing the double-precision floating-point datatype of [[IEEE 754-2008]](https://www.w3.org/TR/xmlschema11-2/#ieee754-2008).  That specification does not specify specific [·lexical representations·](https://www.w3.org/TR/xmlschema11-2/#dt-lexical-representation), but does prescribe requirements on any [·lexical mapping·](https://www.w3.org/TR/xmlschema11-2/#dt-lexical-mapping) used.  Any [·lexical mapping·](https://www.w3.org/TR/xmlschema11-2/#dt-lexical-mapping) that maps the [·lexical space·](https://www.w3.org/TR/xmlschema11-2/#dt-lexical-space) just described onto the [·value space·](https://www.w3.org/TR/xmlschema11-2/#dt-value-space), is a function, satisfies the requirements of [[IEEE 754-2008]](https://www.w3.org/TR/xmlschema11-2/#ieee754-2008), and correctly handles the mapping of the literals 'INF', 'NaN', etc., to the [·special values·](https://www.w3.org/TR/xmlschema11-2/#dt-specialvalue), satisfies the conformance requirements of this specification.

Reply:
As far as I am concerned RDF has to have a specific lexical mapping.   There is one given in the XSD document, but it maps some numbers to +INF and -INF.